tests/test_tree.rs

```diff
@@ -49,7 +50,7 @@ fn kv(i: usize) -> Vec<u8> {
 #[test]
 fn it_works() {
     println!("========== initial sets ==========");
-    let t = Tree::new();
+    let t = Arc::new(Tree::new());
```

Comment:
Interesting 🤔!

Haven't seen the file usage. The "poor man's shared memory" approach is also new to me! Thanks for sharing 👍